std/traits.d

```diff
@@ -5346,7 +5346,9 @@ convertible to a string.
  */
 template isConvertibleToString(T)
```

Comment:
I think `isConvertibleToString` is too misleading to expose. If I saw that in a signature I'd expect `string` to be valid as it is, in fact, convertible to `string`.

Would `is(T : string) && !isExactSomeString` work? (even better if `isExactSomeString` could just be `isSomeString`).